### PR TITLE
(FB: 130819) - When copying form, modify name

### PR DESCRIPTION
**Pull request by benrudolph**
When copying a form, this will add ‘Copy of’ around name. This is a simple implementation and does not look at other forms in the app to determine whether the form has been copied before.

http://manage.dimagi.com/default.asp?130819#858895

### CODE REVIEW

File: corehq/apps/app_manager/models.py [hunk 1]
```diff
@@ -3454,6 +3454,9 @@ def _copy_form(self, from_module, form, to_module):
         if 'unique_id' in copy_source:
             del copy_source['unique_id']
 
+        for lang, name in copy_source['name'].iteritems():
+            with override(lang):
+                copy_source['name'][lang] = _('Copy of %(name)s') % { 'name': name }
```
Comment (dimagimon) [inline corehq/apps/app_manager/models.py]: E201 whitespace after '{'

Comment (dimagimon) [inline corehq/apps/app_manager/models.py]: E202 whitespace before '}'

**Comment by benrudolph** [comment]
don't merge yet, i think this will cause issues when upgrading to advanced: https://github.com/benrudolph/commcare-hq/blob/copy-form-name-130819/corehq/apps/app_manager/models.py#L3499

**Comment by esoergel** [comment]
Just added the `Open for review: hold off on merge` tag.  Useful for situations like this.

File: corehq/apps/app_manager/models.py (hunk 1 above)
Comment (esoergel) [inline corehq/apps/app_manager/models.py]: If the string is short and/or there's only one var to sub in, you can often just do this:
```python
_('Copy of %s') % name
```
Also, while both are valid (and I normally wouldn't even comment on it except that you're somewhat new to python), the more modern form of string formatting is using the string's `format` method:
```python
_("Copy of {name}").format(name=name)
# or
_("Copy of {}").format(name)
```
For more, and for format options like alignment/padding or number representation, check out [the docs on string formatting](https://docs.python.org/2/library/string.html#formatspec)

Comment (benrudolph) [inline corehq/apps/app_manager/models.py]: thanks, noted

**Comment by dimagimon** [comment]
:+1: No lint errors found.

**Comment by czue** [comment]
:+1: for tests!

**Comment by czue** [comment]
anyone ever feel like @dimagimon has weakened the :+1: ? i know i do...

**Pull request merged**